static/coffee/controllers.coffee

```diff
@@ -485,7 +485,7 @@ controllers.controller('HomeController', ['$scope', '$controller', 'LabelService
         $scope.labels = labels
       )
     else if tab == 'users'
-      UserService.fetchNonPartner(true).then((users) ->
+      UserService.fetchAll(true).then((users) ->
```

Comment:
This changes the org users tab to include all users across all partner orgs. 

There's certainly an argument for including them on that tab but I think it would be a checkbox that the user can select, and we'd want to add new fields to show which partner each user is in.